SessionUtilitiesKit/General/SNUserDefaults.swift

```diff
@@ -0,0 +1,112 @@
+import Foundation
```

Comment:
This type seems to have been added back somehow (we should just have a `UserDefaultsType` file now which contains this logic)